fedmsg.d/fas_credentials.py

```diff
@@ -1,8 +1,6 @@
-import getpass
```

Comment:
But doesn't it now breaks running the app itself?

Before you would launch the app, it would ask you for your username and password and work. Now we have to edit this file first or see it crash. Could there be a way to specify a file for the tests?